Assets/PatchKit Patcher/Patcher.unity

```diff
@@ -1,19 +1,19 @@
 %YAML 1.1
```

Comment:
There is a huge amount of changes for this file, it is caused by the fact that it's the first modification of the scene since the upgrade of Unity version, am I right?

Reply:
The last change to the scene was on 2/3/2020, only the app_secret of the sample game has changed. The update project to 2018.4.2f1 was on 21/06/2019.